napari/_vendor/qt_json_builder/qt_jsonschema_form/form.py

```diff
@@ -112,10 +113,13 @@ def create_widget(
         if "enum" in schema:
             default_variant = "enum"
 
+        
         widget_variant = ui_schema.get('ui:widget', default_variant)
         widget_cls = self.widget_map[schema_type][widget_variant]
 
         widget = widget_cls(schema, ui_schema, self)
+       
+        widget.setDescription(description)
```

Comment:
This might break for the other widgets if they do not define this.
]Maybe a try/except, or adding this method on all the other widgets?

Reply:
yes, you are right.  this will break other widgets.  I had to add a setDescription for another just to get it work.  but, I can change to try/except.